### PR TITLE
Resolve #12 Expanded default item

### LINKED ISSUE #12: Expanded default item
Feature request: user would like to have one accordion item expanded by default as the page has loaded.

### PR DESCRIPTION
**Pull request by ain**

### CODE REVIEW

**Comment by ain** [comment]
@davleh could you please review. Thanks!

File: src/iptools-jquery-accordion.js [hunk 1]
```diff
@@ -92,7 +92,9 @@
     init: function() {
 
       this.$panels = this.$element.children(selectorFromClass(this.settings.panelClass));
-      var $activePanels = this.$panels.filter(selectorFromClass(this.settings.panelActiveClass));
+      var $activePanels = this.$panels
+        .filter(selectorFromClass(this.settings.panelActiveClass))
+        .add(this.$panels.filter('[data-accordion-expanded="true"]'));
```
Comment (davleh) [inline src/iptools-jquery-accordion.js]: @ain I would call for naming the data attribute "data-panel-active" or "data-accordion-panel-active" as it is only the panel that will be expanded (not the whole accordion) and to be consistent with the class naming.

**Comment by davleh** [comment]
Only one little remark... Everything else is fine.

File: src/iptools-jquery-accordion.js (hunk 1 above)
Comment (ain) [inline src/iptools-jquery-accordion.js]: Agreed Re consistency. What about `data-accordion-panel-activated` to have better semantics in terms of readability?

Comment (davleh) [inline src/iptools-jquery-accordion.js]: Yes, even better!

Comment (ain) [inline src/iptools-jquery-accordion.js]: :+1:

**Comment by ain** [comment]
Thanks for the prompt reviews @davleh!

**Pull request merged**